packages/client/src/views/SysAdmin/Config/Certificates/Certificates.tsx

```diff
@@ -327,15 +322,13 @@ class CertificatesConfigComponent extends React.Component<Props, State> {
         svgFilename,
         user,
         status,
-        event,
-        [
-          {
-            query: GET_ACTIVE_CERTIFICATES
-          }
-        ]
+        event
       )
       if (res && res.data) {
         this.setState({ imageUploading: false })
+        this.props.updateOfflineCertificate(
+          res.data.createOrUpdateCertificateSVG
```

Comment:
Can we improve the type for `res` so that it's not any?

Reply:
Improved